src/act.offensive.cpp

```diff
@@ -1263,6 +1263,11 @@ ACMD(do_rescue) {
         return;
     }
 
+    if (GET_SKILL(ch, SKILL_BERSERK) && !EFF_FLAGGED(ch, EFF_BERSERK)) {
```

Comment:
I feel berserkers should auto-rescue... even if they don't want to :P
Having to be angry to rescue someone seems weird.

Reply:
What's the code to know if something is fighting?  Since Berserkers can't flee combat either, I think it's not the greatest idea to force them to rescue, but that's just me.

I made it so they have to be berserking to rescue to keep it in the playstyle milieu.  We could easily drop this whole section and just give them the rescue skill like all the other classes.

Reply:
Yeah, being able to rescue a friend is definitely not what I would call 'berserker' behavior.  (In fact, I would be more worried about them attacking friends according to lore.)  I wonder how D&D handles a raging berserker in respect to friends.

Either way, I'm ok approving it.  It doesn't follow the lore in my head, but as a tank class, they need some way to rescue the squishes.  I would even be inclined just to give them the rescue skill since they're still a warrior, even if they're not currently raging.  They can still stand between the beast and a friend.  Also, we don't want to make them less viable tanks because they can't instant rescue on harder content.

Reply:
In D&D when a character rages it just increases their damage while reducing their defensive capabilities.  They don't suddenly become forced to attack allies.

I think by lowering the rage threshold to 100, we've got the limit low enough they should be able to berserk pretty easily.  I'll be leveling another character to find out how it works in the field.